ocaml/idl/datamodel_pool.ml

```diff
@@ -788,7 +810,9 @@ open Datamodel_types
         ; add_to_guest_agent_config
         ; remove_from_guest_agent_config
         ; rotate_secret
-        ; set_repository
+        ; set_repositories
+        ; add_repositories
+        ; remove_repositories
```

Comment:
I think these functions should be called `add_repository` and `remove_repository` respectively, since they only deal with a single repository